.github/workflows/dependabot-post-steps.yaml

```diff
@@ -0,0 +1,35 @@
+name: Run custom steps on Dependabot PRs
+
+on:
+  pull_request:
```

Comment:
Maybe also ad filter when changes are touching the lock file? As there's no need for go updates